ckanext/dgu/theme/templates/page.html

```diff
@@ -15,18 +15,16 @@
 #}
 {%- block content %}
     {% block maintag %}<div role="main">{% endblock %}
-      <div id="content">
-        {% block main_content %}
```

Comment:
ok, i get it, you were talking about page.html.
I did not remove main_content, i move 'toolbar' out of it; 'main_content' block is below. From my understanding of the CKAN templates, this is the best way to have proper layout of the toolbar and keep the original styling.